content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
 
-  # default: vendor
-  vendor_directory: vendor
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER="composer"
+{{< /code/copyable >}}
 
-  # directory where composer.json can be found
-  # default is app root
-  json_path: composer
+### Set Composer authentication
+To set up authentication for Composer, the Composer-native environment variable
+`COMPOSER_AUTH` can be set as a JSON-formatted object containing objects
+defined in the [Composer
+docs](https://getcomposer.org/doc/03-cli.md#composer-auth).
 
-  # if included, will run `composer global` with specified arguments
-  install_global: ["list", "of", "install", "options"]
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER_AUTH='{"http-basic": <some-creds>}'
 {{< /code/copyable >}}
 
-## Select a Web Server
+## Select a web server
 
 The PHP buildpack supports the use of 3 different web servers:
 
- - PHP built-in Web Server
- - Apache HTTP Web Server
+ - PHP Built-in Web Server (default)
+ - Apache HTTPD Web Server
  - Nginx Web Server
 
-You can configure the webserver using `buildpack.yml` as follows:
+You can configure which web server to use by setting the `BP_PHP_SERVER` environment variable at build-time.
+The setting options are `php-server`, `httpd`, `nginx`, with the default value of `php-server` if unset.
+{{< code/copyable >}}
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_PHP_SERVER="php-server"
+{{< /code/copyable >}}
+
+#### Deprecated: buildpack.yml configuration
+Note that specifying any configurations through `buildpack.yml` configuration
+will be deprecated in PHP language-family buildpack v1.0.0.
+To migrate from using `buildpack.yml` please set the environment variable
+equivalents instead.
+
+## Set server configuration
+If you're using `httpd` or `nginx`, a suitable `httpd.conf` or `nginx.conf`
+will be generated for you by the buildpack and made available in a layer.
+Check out the [PHP Reference documentation]({{< ref
+"docs/reference/php-reference" >}}) for an enumeration of the defaults set for
+`httpd` and `nginx`.
+
+Additional configuration can also be provided via environment variables.
+
+### Provide your own web server configuration file
+If either `httpd` or `nginx` are the selected web server, users can provide
+their own configurations in the form of a server-specific configuration file.
+
+#### Provide `httpd` specific configuration
+In the `httpd` case, user-included configuration can be placed be found in the
+application source directory under `<app-directory>/.httpd.conf.d/*.conf`.
+
+#### Provide `nginx` specific configuration
+In the `nginx` case, user-included configuration can be placed in the
+application source directory under `<app-directory>/.nginx.conf.d/`.
+Server-specific configuration should be inside a file named  `*-server.conf`,
+and HTTP configuration should be inside a file with the naming structure
+`*-http.conf`.
```

Comment:
I'm not sure I understand the distinction between server configuration and HTTP configuration. How  do I determine which one to use?

Reply:
I'll link out. I don't think it's my place to describe.